wrappers/matlab/option.m

```diff
@@ -73,6 +73,7 @@
         sensor_mode                     (70)
         emitter_always_on               (71)
         thermal_compensation            (72)
-        count                           (73)
+        hum_temperature                 (73)
```

Comment:
The numbers are not consistent - add the missing [73-80] range

Reply:
Done